test/debt/IncurDebt.js

```diff
@@ -0,0 +1,944 @@
+const chai = require("chai");
+const { assert, expect } = require("chai");
+const { olympus } = require("../utils/olympus");
+const { solidity } = require("ethereum-waffle");
+const { increase } = require("../utils/advancement");
+const { fork_network } = require("../utils/network_fork");
+const impersonateAccount = require("../utils/impersonate_account");
+
+chai.use(solidity);
+
+const gOhmHolderAddress = "0xD3D086B36d5502122F275F4Bc18e04c844Bd6E2e";
+const sOhmHolderAddress = "0xB4e168232ac61b49053f263F57920f9BF38856C2";
+
+describe("IncurDebtV1", async () => {
+    let user,
+        limit,
+        amount,
+        staking,
+        governor,
+        treasury,
+        ohm_token,
+        gohm_token,
+        sohm_token,
+        gOhmHolder,
+        sOhmHolder,
+        IncurDebtV1,
+        incurDebtV1,
+        amountInGOHM,
+        amountInSOHM;
+
+    beforeEach(async () => {
+        await fork_network(14434969);
```

Comment:
SHould not do this here. FOrking mainnet makes the tests extremely slow. If you're just using it to get an account with sOHM and gOHM, just create accounts and send them mock  tokens.

Reply:
Using it to test real-life instance, about it being slow that happens when you’re running it for the first time after that the speed is quite decent.